Dockerfile

```
@@ -1,143 +1,108 @@
 
-RUN apk add --no-cache bash
-RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.alpine.sh' | bash
-RUN apk add symfony-cli
-RUN apk add --no-cache make
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    bash \
+    curl \
+    make \
+    && rm -rf /var/lib/apt/lists/*
 
+RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.deb.sh' | bash
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)